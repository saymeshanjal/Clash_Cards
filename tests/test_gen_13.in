51
10 22 4
12 30 3
6 76 4
6 0 4
8 40 1
12 66 5
10 94 5
4 6 5
6 0 1
14 20 5
14 5 2
2 10 5
12 45 1
8 71 3
14 2 4
6 71 2
4 68 1
6 37 3
16 7 5
2 33 2
6 51 2
8 7 3
14 72 5
4 96 1
2 10 1
2 84 2
6 97 2
16 8 4
14 7 4
12 19 4
10 90 1
4 37 5
2 95 3
10 40 1
16 72 3
12 85 4
8 43 2
10 94 2
8 92 4
4 41 5
10 62 5
8 11 2
14 47 4
4 19 4
2 46 4
4 36 4
14 59 2
4 31 2
10 46 1
14 11 3
2 8 2