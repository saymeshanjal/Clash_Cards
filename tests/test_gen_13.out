2 187 63
180 744 0
1 14 5
0 20 0
35 35 0
219 705 0
45 207 0
0 6 0
0 20 0
56 3376 0
120 1128 2184
0 2 0
462 462 0
27 43 0
0 3432 0
5 6 9
3 3 0
6 14 0
9 12861 0
0 1 1
5 6 9
7 63 0
987 2445 0
3 3 0
1 1 0
0 1 1
5 6 9
1 11673 1196
1 3032 399
14 821 89
126 126 0
0 6 0
0 2 0
126 126 0
5927 6943 0
208 441 275
20 21 29
76 77 99
8 42 20
0 6 0
45 207 0
2 13 55
128 1806 1498
0 5 1
0 2 0
0 5 1
1055 1057 1320
1 2 3
126 126 0
249 3183 0
0 1 1
